packages/jtex/src/tex/export.ts

```diff
@@ -1,16 +1,29 @@
 import path from 'node:path';
 import type MystTemplate from 'myst-templates';
 import { createCommand } from '../utils.js';
+import which from 'which';
+
+export function isLatexmkAvailable() {
+  return which.sync('latexmk', { nothrow: true });
+}
+
+export function isTectonicAvailable() {
+  return which.sync('tectonic', { nothrow: true });
+}
 
 export function pdfTexExportCommand(
```

Comment:
I think let's remove the duplication here by moving this command to `myst-cli/src/build/pdf`.

Reply:
Great.
The only thing regarding the duplicated functions is that I cannot import them from `myst-cli/src/build/pdf/utils.ts` to `jtex/src/tex/export.ts`.

Reply:
I was thinking to move the whole file over, but I see that is a bit complex, and we probably want to do it for typst at the same time.

I am happy to come in after and do a quick refactor, I also want to talk to @fwkoch to get a sense check on moving those around.

Have you tested the functionality locally?

Reply:
Yes, it works on my local machine. Cloned the forked repo and followed your contribution guidelines for developers.

Reply:
Fantastic - I will give it a go today or tomorrow and we can get this released early next week. Thanks again for your help in the PR, excited to try out tectonic and stop recommending 5gb downloads ... !

Reply:
😄 
My pleasure. Yes, it's still relatively unknown, but it's such a step forward for local LaTeX compiling.

Reply:
Regarding the duplication - it would be nice to eliminate... that would mean "creating the command" (currently in `jtex`) and "invoking the command" (currently in `myst-cli`) in one single place. Definitely a smaller refactor to move "creation" to `myst-cli`, but it could also be interesting if part of the scope of `jtex` was to know how to create and invoke all the various rendering engines. MyST could then just say "jtex, build me a pdf using this template" hmm...

Anyway - the duplication here is relatively minor; I don't think we need to block this PR with a big refactor.